pkg/hubble/parser/threefour/parser_test.go

```diff
@@ -1430,9 +1486,10 @@ func TestDecode_DropNotify(t *testing.T) {
 		{
 			name: "drop_unknown",
 			event: monitor.DropNotify{
-				Type: byte(monitorAPI.MessageTypeDrop),
-				File: 2,
-				Line: 42,
+				Type:    byte(monitorAPI.MessageTypeDrop),
+				File:    2,
+				Line:    42,
+				Version: monitor.DropNotifyVersion2,
```

Comment:
Pushed these changes, need to update DropNotify version here, otherwise wrong DataOffset (then wrong decoding).